spec/presenters/document_presenter_spec.rb

```diff
@@ -267,4 +267,45 @@
       expect(subject.breadcrumbs).to eq(breadcrumbs_array)
     end
   end
+
+  describe "organisations" do
+    let(:links) do
+      [
+        OpenStruct.new(content_id: SecureRandom.uuid),
+        OpenStruct.new(content_id: SecureRandom.uuid),
+      ]
+    end
+
+    context "when the finder has organisations links" do
+      before do
+        finder.links = OpenStruct.new(
+          organisations: links,
+        )
+      end
+
+      it "it returns the finder links" do
+        expect(subject.organisations).to be(finder.links.organisations)
+      end
+    end
+
+    context "when the finder has links but no organisations ones" do
+      before do
+        finder.links = OpenStruct.new(
+          different_links: links,
+        )
+      end
+
+      it "returns an empty array" do
+        expect(subject.organisations).to eq([])
```

Comment:
does that only match enumerables? if so that'll be nicer - I checked that it doesn't match nil so that's good.